todos.js

```diff
@@ -10,12 +10,17 @@ const renderJson = json => {
   const userTasks = json
     .map(
       userTask =>
```

Comment:
you can deconstruct the `userTask` here

Reply:
I second this

Reply:
Feel free to ask about destructuring if you're not sure

Reply:
https://wesbos.com/destructuring-objects/